Geometry/GEMGeometry/src/ME0Chamber.cc

```diff
@@ -78,3 +78,23 @@ const ME0EtaPartition* ME0Chamber::etaPartition(int isl) const {
   }
   return 0;
 }
+
+float ME0Chamber::computeDeltaPhi(const LocalPoint& position, const LocalVector& direction ) const {
```

Comment:
the dPhi used currently is the track phi minus the segment phi.
I don't think this give you more information compared to just the toGlobal(direction), because you are just scaling this by chamber size.